pkg/tagger/utils/expiry_test.go

```diff
@@ -0,0 +1,75 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2017 Datadog, Inc.
+
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestUpdate(t *testing.T) {
+	expire := &Expire{
```

Comment:
let's just call a `NewExpire` to make sure the constructor is OK too

Reply:
👌